modules/ngx_http_lua_upstream/.travis.yml

```diff
@@ -0,0 +1,73 @@
+sudo: required
```

Comment:
i think you need to add this automatic CI/CD task into the .github/

Reply:
I added new modules to CI of tengine.
Maybe its what we really need?
![image](https://github.com/alibaba/tengine/assets/3404496/f0a826d5-0583-460b-ab20-5607685d9c8e)

[https://github.com/alibaba/tengine/pull/1882/commits/9556ff8e22aa0319754ede9720f1cbcb127e650f](url)

Reply:
![image](https://github.com/alibaba/tengine/assets/3404496/5e464bb7-8092-4c6a-91bf-8a1440d64539)

Reply:
Already doen